lib/perl/Genome/Config/AnalysisProject/Command/Release.pm

```diff
@@ -46,15 +46,27 @@ sub execute {
 sub __errors__ {
     my $self = shift;
     my @errors = $self->SUPER::__errors__(@_);
-    for my $status (map{$_->status} $self->analysis_projects){
+
+    for my $anp ($self->analysis_projects) {
+        my $status = $anp->status;
         unless(grep{$_ eq $status} ("Pending", "Hold", "In Progress")){
             push @errors, UR::Object::Tag->create(
                 type => 'error',
                 properties => ['analysis_projects'],
-                desc => "Can't release analysis project with status: $status"
+                desc => "Can't release analysis project " . $anp->__display_name__ . " with status: $status"
+            );
+        }
+
+        my $config = $anp->environment_config_dir;
+        unless ($config) {
```

Comment:
Yeah, validating the contents of the config seems like a losing battle at this point. I was just debating seeing if there was a non-empty file of some type there, but if that's already being done in the accessor, I think its fine "as-is."